source/retryable-writes/tests/insertOne-errorLabels.yml

```diff
@@ -0,0 +1,80 @@
+runOn:
+    -
+        minServerVersion: "4.3.0"
+        topology: ["replicaset", "sharded"]
+
+data: []
+
+tests:
+    -
+        description: "InsertOne succeeds after RetryableWriteError from server"
```

Comment:
After reviewing other tests in this file, I think this may be the only one that justifies keeping `insertOne-errorLabels.yml`.